Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -678,9 +678,13 @@ def wrapper(*args, **kwargs):
             print_debug_msg(f'Attempting to update context data after version {version} with retry {retries}')
             new_context_data, new_version = get_integration_context_with_version()
             if new_version == version:
-                set_to_integration_context_with_retries(context_data)
-                context_was_set = True
-                print_debug_msg(f'Updated integration context after version {version} in retry {retries}.')
+                try:
+                    set_to_integration_context_with_retries(context_data, max_retry_times=1)
+                    context_was_set = True
+                    print_debug_msg(f'Updated integration context after version {version} in retry {retries}.')
+                except Exception as e:
+                    if 'Max retry attempts exceeded' in str(e):
+                        continue
```

Comment:
add here retries = retries + 1 ( or move the retries = retries + 1)

Reply:
Another question that came to mind: you dont handle else (where the exception is not with 'Max retry attempts exceeded'. What do you think we would want to do there? re-raise the exception perhaps?